qiskit/visualization/utils.py

```diff
@@ -129,6 +130,9 @@ def get_param_str(op, drawer, ndigits=3):
     if not hasattr(op, "params") or any(isinstance(param, np.ndarray) for param in op.params):
         return ""
 
+    if isinstance(op, ControlFlowOp):
+        return ""
+
```

Comment:
If it's simple to do, it might be nice to still return the condition for `IfElseOp` and `WhileLoopOp`, and the parameter and indexset for `ForLoopOp`.  If it's not absolutely trivial, let's just keep what you've written though, and merge this as-is.

Reply:
Good point. I thought about this a bit, but we moved `condition` out of `params` and into `Instruction.condition` as part of #7123 (though the plan is for them to eventually move back) and the parameter name (especially if auto-generated) and indexset alone don't provide much context in the case of a `ForLoopOp` so I think it's okay to leave these as opaque in the drawers for now and we can revisit better ways to visualize.

Reply:
Yep, I buy that argument for now.  Let's merge this as-is.